packages/wrangler/src/index.tsx

```diff
@@ -229,6 +229,10 @@ export async function main(argv: string[]): Promise<void> {
               {
                 name: "worker",
```

Comment:
This should also match the name in the `wrangler.toml`.

Reply:
I can add that 😄